test/libyul/yulOptimizerTests/forLoopInitRewriter/simple.yul

```diff
@@ -0,0 +1,21 @@
+{
+  let random := 42
+  for { let a := 1 } iszero(eq(a, 10)) { a := add(a, 1) } {
```

Comment:
Please also add a test that uses multiple statements here.

Reply:
done that in the `complex_pre.yul`